### DISCUSSION TITLE
Delete all Photos from Person

### DISCUSSION THREAD

**Discussion by BoBBer446** [nextcloud/recognize#1053] [Q&A]
Is there a feature in Nextcloud Recognize to select and delete all photos of a specific person at once? Currently, it seems necessary to select each photo individually, which is time-consuming. Are there any shortcuts or batch processing methods available to speed up this process?

What does this meaning ?
![image](https://github.com/nextcloud/recognize/assets/10163536/21d066c1-5d01-4ab0-9638-1b0df2bc737e)

**Comment by Olen** [comment]
"Merge with different person" moves all the Photos from Person 1 (the currently selected person) to another person.
"Remove Person" will untag all the Photos in this cluster.
